services/api/src/models/role.js

```diff
@@ -0,0 +1,17 @@
+const { createSchema } = require('../utils/schema');
+const mongoose = require('mongoose');
+const definition = require('./definitions/role.json');
+const { validContexts, createDefaultPermissions, validatePermissions } = require('../utils/permissions');
+
+const permissions = createDefaultPermissions();
+definition.attributes.context.enum = validContexts;
+definition.attributes.permissions = permissions;
```

Comment:
wondering if this could change the reference to "definitions/role.json" dont recall how require works ... 
but general a bad pattern